.changeset/fifty-apples-walk.md

```diff
@@ -0,0 +1,21 @@
+---
+"@builder.io/sdk-angular": major
+"@builder.io/sdk-react-nextjs": major
+"@builder.io/sdk-qwik": major
+"@builder.io/sdk-react": major
+"@builder.io/sdk-react-native": major
+"@builder.io/sdk-solid": major
+"@builder.io/sdk-svelte": major
+"@builder.io/sdk-vue": major
```

Comment:
one small caveat: Qwik, NextJS and Angular SDKs are still in beta `v0.x`, so for those we will mark this as a `minor` update instead of `major` to avoid them becoming `v1.0`